integration/test/chart/basic/basic_test.go

```diff
@@ -12,11 +13,51 @@ import (
 	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/helmclient/v3/pkg/helmclient"
 	"github.com/giantswarm/microerror"
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
 
 	"github.com/giantswarm/chart-operator/v2/integration/key"
 )
 
+const (
+	configMapName = "test-app-configmap"
+	secretName    = "test-app-configmap"
+
+	configmapValue = `
+v1:
+  username: admin
+  host:
+    url: quay.io
+  memory_in_gb: 0
+  threshold: 2.17
+replicas: 3
+`
+	secretValue = `
+v1:
+  username: admin
+  host: 
+    secret: 
+      authToken: xer32wnq
+  githubToken: nnbhwk1dk
+  memory_in_gb: 3.14
+`
+
+	mergedValue = `
```

Comment:
I made values as complex as I could. @rossf7 PTAL

Reply:
These look good. Most types are covered. What about adding a boolean? Maybe also some quoted strings? 

Basically anything in YAML that is a bit weird. Which is most things. 😄

Reply:
Good call, I would add both cases.